src/common/low_precision_transformations/include/low_precision/common/precisions_restriction.hpp

```diff
@@ -19,35 +19,41 @@ namespace ngraph {
 namespace pass {
 namespace low_precision {
 
+class Ports : public std::vector<size_t> {
```

Comment:
Do we really need separate wrapper for std::vector<size_t>. Especially in header. Seems like it is used only inside PrecisionsRestriction class

Reply:
The only reason is to overload constructor from size_t.
Thanks to that we can save current PrecisionRestriction setting way (with one port as size_t for each restriction)
```
PrecisionsRestriction::create<ngraph::opset1::GroupConvolution>({
    {0, {ngraph::element::u8}},
    {1, {ngraph::element::i8}}
}),
```
If we hadn't used this wrapper, the std::vector<size_t>(size_t) constructor would have been called. E.g. for restriction `{0, {ngraph::element::u8}}` empty ports vector would have been created. However, expected ports vector is vector<size_t>{0}.
If we remove this wrapper, all existing restrictions in plugins and code snippets should be changed. E.g. restriction above should be changed to
```
PrecisionsRestriction::create<ngraph::opset1::GroupConvolution>({
    {{0}, {ngraph::element::u8}},
    {{1}, {ngraph::element::i8}}
}),
```

@dmitry-gorokhov Is it a good idea to change all PrecisionRestrictions declarations? Or maybe you could offer another way to keep the current PrecisionRestrictions declaration interface and not introduce any wrappers?

Reply:
Seems like you can just add overload for create method which internally will cast to new PrecisionsByPorts representation

Reply:
When lets update creation code on plugins side. I would prefer to avoid introduction of new class required just to maintain old semantics (given this is not public API).

Reply:
Okay, done, thanks